clients/frontend-authorization/src/frontend-authorization-client.ts

```diff
@@ -3,9 +3,13 @@
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
 export * from "./FrontendAuthorizationClient";
+export * from "./FrontendAuthorizationClientLoggerCategory";

```

Comment:
thanks! just hit this myself. 

will this be backported to 2.19?

Reply:
That makes sense. I'll backport it.